packages/preferences/src/browser/folder-preference-provider.ts

```diff
@@ -36,9 +37,10 @@ export class FolderPreferenceProvider extends AbstractResourcePreferenceProvider
 
     readonly folderUri: URI;
 
+    @inject(WorkspaceService) protected readonly workspaceService: WorkspaceService;
+
     constructor(
-        @inject(FolderPreferenceProviderOptions) protected readonly options: FolderPreferenceProviderOptions,
-        @inject(FileSystem) protected readonly fileSystem: FileSystem
+        @inject(FolderPreferenceProviderOptions) protected readonly options: FolderPreferenceProviderOptions
```

Comment:
@akosyakov I am breaking the constructor. and as you can see, it is not a necessary change. 
are you ok with it ?

Reply:
after a second thought I removed constructor completely @akosyakov